src/ui/YouTube/YouTubeThumbnail.ts

```diff
@@ -127,13 +137,15 @@ export class YouTubeThumbnail extends Component {
 
   private openYoutubeIframe() {
     // need to put iframe inside div : iframe with position absolute and left:0, right : 0 , bottom: 0 is not standard/supported
-    let iframe = $$('iframe'),
-      div = $$('div');
-    iframe.setAttribute('src', 'https://www.youtube.com/embed/' + this.extractVideoId() + '?autoplay=1');
-    iframe.setAttribute('allowfullscreen', 'allowfullscreen');
-    iframe.setAttribute('webkitallowfullscreen', 'webkitallowfullscreen');
-    iframe.setAttribute('width', '100%');
-    iframe.setAttribute('height', '100%');
+    const iframe = $$('iframe', {
+      src: `https://www.youtube.com/embed/${this.extractVideoId()}?autoplay=1`,
+      allowfullscreen: 'allowfullscreen',
+      webkitallowfullscreen: 'webkitallowfullscreen',
```

Comment:
I wonder if it would be possible to use a 3rd party package for cross-browser compatibility similar to CSS. So when we add `allowfullscreen`, it will add the webkit, moz equivalents.

Reply:
I think i'll simply remove the vendor specific attribute, it's been supported by all major browsers since 2013... ;)